test/functional/mempool_accept.py

```diff
@@ -99,21 +88,15 @@ def run_test(self):
         )
 
         self.log.info('A final transaction not in the mempool')
-        coin = coins.pop()  # Pick a random coin(base) to spend
-        output_amount = Decimal('0.025')
-        raw_tx_final = node.signrawtransactionwithwallet(node.createrawtransaction(
-            inputs=[{'txid': coin['txid'], 'vout': coin['vout'], "sequence": 0xffffffff}],  # SEQUENCE_FINAL
-            outputs=[{node.getnewaddress(): output_amount}],
-            locktime=node.getblockcount() + 2000,  # Can be anything
-        ))['hex']
-        tx.deserialize(BytesIO(hex_str_to_bytes(raw_tx_final)))
-        fee_expected = coin['amount'] - output_amount
+        raw_tx_final = miniwallet.send_self_transfer(from_node = node, send_tx = False)
```

Comment:
why is this no longer passing SEQUENCE_FINAL, etc...?

Reply:
Thats because the default is set to `0xffffffff` in `send_send_transfer` parameter

Reply:
So if the default changes, the test will fail? If so, I'd prefer to set it here to avoid a changed default breaking the test.

Reply:
Got it! Thank you